src/runtime/contrib/tensorrt/tensorrt_runtime.cc

```diff
@@ -64,7 +64,8 @@ class TensorRTRuntime : public JSONRuntimeBase {
                            const Array<String>& const_names)
       : JSONRuntimeBase(symbol_name, graph_json, const_names),
         use_implicit_batch_(true),
-        max_workspace_size_(size_t(1) << 30) {}
+        max_workspace_size_(size_t(1) << 30),
+        highest_batch_size_(-1) {}
```

Comment:
Better to make it consistent.
```suggestion
        max_batch_size_(-1) {}
```